example/subgraph_op/imagenet_inference.py

```diff
@@ -0,0 +1,166 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
```

Comment:
Could you write a simple README for this file? How to run this test? How can users play with this test?

Reply:
You can use the following command to try it. Please remember to change `ctx` passed to the function `score` into `mx.cpu()` if you are running on a cpu. It's just a preliminary end-to-end run-through for proof of concept. More changes are coming.
```python
python --model=imagenet1k-resnet-152 --dataset=data --num-inference-batches=10
```